src/boot/efi/random-seed.c

```diff
@@ -14,15 +14,28 @@
 
 #define EFI_RNG_GUID &(const EFI_GUID) EFI_RNG_PROTOCOL_GUID
 
+struct linux_efi_random_seed {
+      uint32_t size;
+      unsigned char seed[];
+};
```

Comment:
6 char indenting? that's new...

Reply:
unsigned char → uint8_t

Reply:
I've got a patch brewing to change systemd formatting style to 3 chars for the first level, 5 for the second, 7 for the third, 11 for the forth, 13 for the fifth, 17 for the sixth. Should be a hoot.

Reply:
hmm, you are joking, but there was a project like that. was it gpsd? (or was that the one where they never used dynamic memory? something was totally nuts about that)